jbpm-wb-common/jbpm-wb-common-api/src/main/java/org/jbpm/workbench/common/preferences/ManagePreferences.java

```diff
@@ -104,4 +110,12 @@ public boolean isShowTaskCommentsAtWorkTab() {
     public void setShowTaskCommentsAtWorkTab(boolean showTaskCommentsAtWorkTab) {
         this.showTaskCommentsAtWorkTab = showTaskCommentsAtWorkTab;
     }
+
+    public String getProcessInstanceDiagramErrorNodeBorderColor() {
```

Comment:
I would suggest moving these methods after `setProcessInstanceDiagramActiveNodeBorderColor` to keep similar methods next to each other.